webservices/legal_docs/es_management.py

```diff
@@ -67,6 +67,7 @@
         },
         "document_date": {"type": "date", "format": "dateOptionalTime"},
         "url": {"type": "text"},
+        "fileno": {"type": "text"},
```

Comment:
Sometimes the filenames can have underscores or letters in them, so they're not always numeric for every occurrence. Since we'll primarily be using the filename as a reverse lookup to get the MUR number using an existing old URL, validation may not be necessary.